src/main/java/mate/academy/model/Ticket.java

```diff
@@ -47,11 +55,12 @@ public void setMovieSession(MovieSession movieSession) {
 
     @Override
     public String toString() {
-        return "Ticket{"
-            + "id=" + id
-            + ", movieSession=" + movieSession
-            + ", user=" + user
-            + '}';
+        return System.lineSeparator()
```

Comment:
It is not good idea to add lineSeparator in `toString()` here . You can use `toString()` later for cases where no need start from the new line.